From f77654b8472a2eccd44260c040483418a49e3b99 Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.io>
Date: Thu, 24 Aug 2017 17:47:06 +0800
Subject: [PATCH 85/98] ARM: sun8i: r40: add GMAC RGMII pinmux

The GMAC of Allwinner R40 SoC is capable of RGMII PHY mode.

Add a pinmux node for it.

Signed-off-by: Icenowy Zheng <icenowy@aosc.io>
---
 arch/arm/boot/dts/sun8i-r40.dtsi | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm/boot/dts/sun8i-r40.dtsi b/arch/arm/boot/dts/sun8i-r40.dtsi
index 32648765319a..f71a358971f6 100644
--- a/arch/arm/boot/dts/sun8i-r40.dtsi
+++ b/arch/arm/boot/dts/sun8i-r40.dtsi
@@ -291,6 +291,16 @@
 			#interrupt-cells = <3>;
 			#gpio-cells = <3>;
 
+			gmac_rgmii_pins: gmac-rgmii-pins {
+				pins = "PA0", "PA1", "PA2", "PA3", "PA4",
+				       "PA5", "PA6", "PA7", "PA8", "PA10",
+				       "PA11", "PA12", "PA13", "PA15",
+				       "PA16";
+				function = "gmac";
+				drive-strength = <30>;
+				bias-pull-up;
+			};
+
 			i2c0_pins: i2c0-pins {
 				pins = "PB0", "PB1";
 				function = "i2c0";
-- 
2.13.6

